packages/create-amplify/src/package-manager-controller/yarn_classic_package_manager_controller.ts

```diff
@@ -17,11 +17,15 @@ export class YarnClassicPackageManagerController extends PackageManagerControlle
     this.initDefault = ['init', '--yes'];
   }
 
+  initializeTsConfig = async (targetDir: string) => {
+    await this.addTypescript(targetDir);
+    await this.initializeTsConfig(targetDir);
```

Comment:
```suggestion
    await super.initializeTsConfig(targetDir);
```

Reply:
> Class field 'initializeTsConfig' defined by the parent class is not accessible in the child class via super.

☹️

Reply:
Can you see if using non-arrow notation in the parent class definition fixes that?

If this works it's fine, but it seems like this would lead to infinite recursion.

Reply:
Yes, non-arrow function works!